src/BenchmarkTools.jl

```diff
@@ -2,11 +2,8 @@ module BenchmarkTools
 
 using JSON
 using Base.Iterators
-
-using Statistics
```

Comment:
We still want to be `using Statistics` here and extending the functions from Statistics; we can simply add `mean` and `median` to the exports, which will reexport the right things.

Reply:
Good suggestion. Updated latest PR.